packages/integration/spec/runtime.spec.js

```diff
@@ -33,16 +33,19 @@ describe(`As developer with
         /*   */ .and()
         /**/ .apply(this);
 
-      await frontUserAction('call hello', this, async (api) => {
-        let failure = false;
-        try {
-          await api.hello();
-        } catch (error) {
-          failure = true;
-        }
-        await this.context.runner.stop();
-        expect(failure).toBe(false);
-      });
+      await frontUserAction()
+        .name('call hello')
+        .context(this)
```

Comment:
Je mettrais .context(this) optionnel et je le passerais plutot au niveau de la méthode frontUserAction() comme paramètre obligatoire.
Ca eviterais d'avoir coder .context(this) à chaque fois.